src/api/app/views/layouts/webui/responsive_ux/_logged_navigation.html.haml

```diff
@@ -0,0 +1,27 @@
+.navbar-collapse.offcanvas-collapse.navbar-dark
+  .navbar-nav
+    .nav.justify-content-end.py-2
+      %button.navbar-toggler{ 'type': 'button', data: { toggle: 'offcanvas' }, aria: { expanded: 'false', label: 'Toggle navigation' } }
+        %i.fas.fa-times
+  %ul.navbar-nav.ml-auto.pt-0.text-nowrap
+    %li.nav-item
+      = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
+        = User.session!.login
+    %li.nav-item
+      - tasks = User.session!.tasks
+      = link_to(my_tasks_path, class: 'nav-link') do
+        - if tasks.positive?
+          %span{ title: 'where an action is requested from you' }
+            = pluralize(tasks, 'Task')
+        - else
+          Tasks
+    %li.nav-item
+      - if User.session!.home_project
+        = link_to(project_show_path(User.session!.home_project), class: 'nav-link') do
+          Home Project
```

Comment:
```suggestion
         Your Home Project
```

Reply:
Maybe also with a :house: icon?

Reply:
We decided to include icons for all the options in the menu. For "Profile" we use the gravatar as you proposed. But, for "Your Home Project" link, we added the project's icon instead of the house (which can be confused with "home/landing page").

<img width="200" src="https://user-images.githubusercontent.com/2581944/73750365-5746a680-475d-11ea-99d8-7f977f6e58fb.png">